components/mdc/TextInput/TextField.svelte

```diff
@@ -56,7 +60,6 @@ const focus = (node) => autofocus && node.focus()
   class="mdc-text-field mdc-text-field--outlined {$$props.class} textfield-radius"
   class:mdc-text-field--no-label={!label}
   class:mdc-text-field--disabled={disabled}
-  class:mdc-text-field--invalid={hasReachedMaxLength}
```

Comment:
Do you still want to flag the TextField as invalid if they've exceeded the max length?

Reply:
yes, I am using `addOrRemoveInvalidClass` to do the same thing but each time value changes otherwise MDC muddles things up.